ports/nrf/Makefile

```diff
@@ -90,9 +90,7 @@ INC += -I../../supervisor/shared/usb
 
 #Debugging/Optimization
 ifeq ($(DEBUG), 1)
-  CFLAGS += -ggdb
-  # You may want to enable these flags to make setting breakpoints easier.
-  CFLAGS += -fno-inline -fno-ipa-sra
```

Comment:
Does -0g do the same thing as no-inline and no-ipa-sra?

Reply:
It disables function inlining and disables ipa-sra.  However, some inlining (specifically, I think this means inlining atomics and functions explicitly marked as inline) still occurs.  I could add -fno-inline if you think it would be preferable.

```arm-none-eabi-gcc -Og -Q --help=optimizers | grep -E 'inline|ipa-sra'
  -finline                    		[enabled]
  -finline-atomics            		[enabled]
  -finline-functions          		[disabled]
  -finline-functions-called-once 	[disabled]
  -finline-small-functions    		[disabled]
  -fipa-sra                   		[disabled]
```

Reply:
(with modern debug information, gdb can show full tracebacks even involving inlined functions)

Reply:
This is fine. Just wondering.